src/storage/AdminProcessor.h

```diff
@@ -96,7 +112,36 @@ class MemberChangeProcessor : public BaseProcessor<cpp2::AdminExecResp> {
     }
 
     void process(const cpp2::MemberChangeReq& req) {
-        UNUSED(req);
+        CHECK_NOTNULL(kvstore_);
+        auto spaceId = req.get_space_id();
+        auto partId = req.get_part_id();
+        LOG(INFO) << "Receive member change for space "
+                  << spaceId << ", part " << partId
+                  << ", add/remove " << (req.get_add() ? "add" : "remove");
+        auto ret = kvstore_->part(spaceId, partId);
+        if (!ok(ret)) {
+            onFinished(to(error(ret)));
+            return;
+        }
+        auto part = nebula::value(ret);
+        auto peer = kvstore::NebulaStore::getRaftAddr(HostAddr(req.get_peer().get_ip(),
+                                                               req.get_peer().get_port()));
+        auto cb = [this, spaceId, partId] (kvstore::ResultCode code) {
+            auto leaderRet = kvstore_->partLeader(spaceId, partId);
+            CHECK(ok(leaderRet));
```

Comment:
is there any plan to replace all CHECK before release a version to customer?maybe it is a big workload to do this because we used too many CHECK. And I think return a failed status to user is a easy way here

Reply:
Good suggestion!  The CHECK here is used to catch bug.  After the software be stable, we could use DCHECK